cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js

```diff
@@ -40,7 +40,9 @@ describe('ui-data-import: A user can delete import logs with "Data import: Can d
           const filePath = numberOfLogsToUpload - 1 === index ? filePathToUpload : emptyFilePathToUpload;
           fileNameToUpload = `C358137autotestFile.${getRandomPostfix()}.mrc`;
           DataImport.uploadFile(filePath, fileNameToUpload);
+          cy.wait(2000);
```

Comment:
please add the reason why you need this waiter

Reply:
I deleted it